Source/WTF/wtf/PlatformHave.h

```diff
@@ -848,6 +848,12 @@
 #define HAVE_ASC_WEBKIT_SPI 1
 #endif
 
+#if PLATFORM(IOS) \
+    || PLATFORM(MAC) \
+    || PLATFORM(VISION)
```

Comment:
What about `IOS_SIMULATOR` or `MACCATALYST`? I suspect these don't work, but wanted to confirm.